editor/plugins/version_control_editor_plugin.cpp

```diff
@@ -95,19 +84,22 @@ VersionControlEditorPlugin *VersionControlEditorPlugin::get_singleton() {
 
 void VersionControlEditorPlugin::popup_vcs_set_up_dialog(const Control *p_gui_base) {
 
-	Size2 popup_size = Size2(400, 100);
-	Size2 window_size = p_gui_base->get_viewport_rect().size;
-	popup_size.x = MIN(window_size.x * 0.5, popup_size.x);
-	popup_size.y = MIN(window_size.y * 0.5, popup_size.y);
+	fetch_available_vcs_addon_names();
+	List<StringName> available_addons = get_available_vcs_names();
+	if (available_addons.size() >= 1) {
 
-	if (get_is_vcs_intialized()) {
+		Size2 popup_size = Size2(400, 100);
+		Size2 window_size = p_gui_base->get_viewport_rect().size;
+		popup_size.x = MIN(window_size.x * 0.5, popup_size.x);
+		popup_size.y = MIN(window_size.y * 0.5, popup_size.y);
 
-		set_up_init_button->set_disabled(true);
-	}
+		_populate_available_vcs_names();
 
-	_populate_available_vcs_names();
+		set_up_dialog->popup_centered_clamped(popup_size * EDSCALE);
+	} else {
 
-	set_up_dialog->popup_centered_clamped(popup_size * EDSCALE);
+		EditorNode::get_singleton()->show_warning(TTR("No VCS plugins are available."), TTR("Error"));
```

Comment:
The plugins were agreed upon to be called 'VCS addons' instead of plugins because they are made to be swappable and removable. This is a nitpick though